modules/postgres/16.2.bcr.1/overlay/BUILD.bazel

```
@@ -0,0 +1,1230 @@
+"""] + select({
+        "@platforms//os:macos": ["/* #undef HAVE_PPOLL */"],
+        "//conditions:default": ["#define HAVE_PPOLL 1"],
+    }) + ["""
+
+/* Define if you have POSIX threads libraries and header files. */
+#define HAVE_PTHREAD 1
+
+/* Define to 1 if you have the `pthread_barrier_wait' function. */
+"""] + select({
+        "@platforms//os:macos": ["/* #undef HAVE_PTHREAD_BARRIER_WAIT */"],
+        "//conditions:default": ["#define HAVE_PTHREAD_BARRIER_WAIT 1"],
+    }) + ["""
+
+/* Define to 1 if you have the `pthread_is_threaded_np' function. */
+"""] + select({
+        "@platforms//os:macos": ["#define HAVE_PTHREAD_IS_THREADED_NP 1"],
+        "//conditions:default": ["/* #undef HAVE_PTHREAD_IS_THREADED_NP */"],
+    }) + ["""
+
+/* Have PTHREAD_PRIO_INHERIT. */
+#define HAVE_PTHREAD_PRIO_INHERIT 1
+
+/* Define to 1 if you have the <readline.h> header file. */
+/* #undef HAVE_READLINE_H */
+
+/* Define to 1 if you have the <readline/history.h> header file. */
+/* #undef HAVE_READLINE_HISTORY_H */
+
+/* Define to 1 if you have the <readline/readline.h> header file. */
+/* #undef HAVE_READLINE_READLINE_H */
+
+/* Define to 1 if you have the `rl_completion_matches' function. */
+/* #undef HAVE_RL_COMPLETION_MATCHES */
+
+/* Define to 1 if you have the global variable 'rl_completion_suppress_quote'.
+   */
+/* #undef HAVE_RL_COMPLETION_SUPPRESS_QUOTE */
+
+/* Define to 1 if you have the `rl_filename_completion_function' function. */
+/* #undef HAVE_RL_FILENAME_COMPLETION_FUNCTION */
+
+/* Define to 1 if you have the global variable 'rl_filename_quote_characters'.
+   */
+/* #undef HAVE_RL_FILENAME_QUOTE_CHARACTERS */
+
+/* Define to 1 if you have the global variable 'rl_filename_quoting_function'.
+   */
+/* #undef HAVE_RL_FILENAME_QUOTING_FUNCTION */
+
+/* Define to 1 if you have the `rl_reset_screen_size' function. */
+/* #undef HAVE_RL_RESET_SCREEN_SIZE */
+
+/* Define to 1 if you have the `rl_variable_bind' function. */
+/* #undef HAVE_RL_VARIABLE_BIND */
+
+/* Define to 1 if you have the <security/pam_appl.h> header file. */
+/* #undef HAVE_SECURITY_PAM_APPL_H */
+
+/* Define to 1 if you have the `setproctitle' function. */
+/* #undef HAVE_SETPROCTITLE */
+
+/* Define to 1 if you have the `setproctitle_fast' function. */
+/* #undef HAVE_SETPROCTITLE_FAST */
+
+/* Define to 1 if the system has the type `socklen_t'. */
+#define HAVE_SOCKLEN_T 1
+
+/* Define to 1 if you have spinlocks. */
+#define HAVE_SPINLOCKS 1
+
+/* Define to 1 if you have the `SSL_CTX_set_cert_cb' function. */
+/* #undef HAVE_SSL_CTX_SET_CERT_CB */
+
+/* Define to 1 if stdbool.h conforms to C99. */
+#define HAVE_STDBOOL_H 1
+
+/* Define to 1 if you have the <stdint.h> header file. */
+#define HAVE_STDINT_H 1
+
+/* Define to 1 if you have the <stdlib.h> header file. */
+#define HAVE_STDLIB_H 1
+
+/* Define to 1 if you have the `strchrnul' function. */
+"""] + select({
+        "@platforms//os:macos": ["#define HAVE_STRCHRNUL 1"],
```

This looks to be an additional change.
```suggestion
        "@platforms//os:macos": ["/* #undef HAVE_STRCHRNUL */"],
```

---

https://buildkite.com/bazel/bcr-presubmit/builds/18317/steps/canvas?sid=0198c4ba-24f7-4ae5-800a-1eef1ff6ce11